tests/src/rules/named.js

```diff
@@ -26,6 +26,7 @@ ruleTester.run('named', rule, {
     test({code: 'import { destructingRenamedAssign } from "./named-exports"'}),
     test({code: 'import { ActionTypes } from "./qc"'}),
     test({code: 'import {a, b, c, d} from "./re-export"'}),
+    test({code: 'import {RuleTester} from "./re-export-node_modules"'}),
```

Comment:
I chose `eslint` as the `node_module` and `RuleTester` as the import because I figured that'd be pretty stable. It's unlikely that export would change, and in the event that it does, we'll need to update this test file quite a bit anyway.